orocos_kdl_vendor/0001-include_project_name.patch

```diff
@@ -1,23 +0,0 @@
-diff --git a/orocos_kdl/src/CMakeLists.txt b/orocos_kdl/src/CMakeLists.txt
-index 954b8fc..306187a 100644
---- a/orocos_kdl/src/CMakeLists.txt
-+++ b/orocos_kdl/src/CMakeLists.txt
-@@ -53,7 +53,7 @@ ENDIF()
- ADD_LIBRARY(orocos-kdl ${LIB_TYPE} ${KDL_SRCS})
- 
- TARGET_INCLUDE_DIRECTORIES(orocos-kdl PUBLIC
--  "$<INSTALL_INTERFACE:include>")
-+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>")
- 
- SET_TARGET_PROPERTIES( orocos-kdl PROPERTIES
-   SOVERSION "${KDL_VERSION_MAJOR}.${KDL_VERSION_MINOR}"
-@@ -79,7 +79,7 @@ INSTALL(TARGETS orocos-kdl
-   ARCHIVE DESTINATION lib${LIB_SUFFIX}
-   LIBRARY DESTINATION lib${LIB_SUFFIX}
-   RUNTIME DESTINATION bin
--  PUBLIC_HEADER DESTINATION include/kdl
-+  PUBLIC_HEADER DESTINATION include/${PROJECT_NAME}/kdl
- )
- 
--INSTALL(FILES ${UTIL_HPPS} DESTINATION include/kdl/utilities)
-+INSTALL(FILES ${UTIL_HPPS} DESTINATION include/${PROJECT_NAME}/kdl/utilities)
```

Comment:
Just to confirm with @sloretz - because the new vendor pattern installs each vendored payload into a separate subdirectory of `/opt/ros/foo/opt/...`, this patch is no longer necessary. Is that correct?

Reply:
Correct that the patch would no longer be necessary. Installing the vendored package to a unique prefix means the include path for the headers it provides are also already unique.

Reply:
Thank you!